benches/coprocessor_executors/mod.rs

```diff
@@ -14,9 +14,13 @@ mod top_n;
 mod util;
 
 fn main() {
+    let mut cpu_set = nix::sched::CpuSet::new();
```

Comment:
🙃 maybe not suitable to leave it here .. It is only for our own benchmark platform.

Reply:
Yes! I forgot to delete it :cry: